ontology/mcro.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: ontology_detail
+id: mcro
+title: Model Card Report Ontology
+contact:
+  email: muamith@utmb.edu
+  github: ProfTuan
+  label: Tuan Amith
+  orcid: 0000-0003-4333-1857
+dependencies:
+- id: iao
+- id: swo
+description: Model card reports are documents detailing transparent metadata information relating to machine learning models. Similar to what we have with drug labels and nutritional labels, the goal of model cards are to communicate relevant information on all aspects of a machine learning model that have undergone any experimentation. However these important reports of the machine learning models are presented in static documents. This work encodes the structure of model card reports and align them to standard OBO Foundry ontologies to help formalize and enrich these documents. The end result is computable model of the model card that can be used to standardize reporting and be integrated in future software tooling (searching and indexing, etc.).
```

Comment:
it doesn't make sense that the description line is longer than the long text, can this be summarized in a single sentence, and the rest added in the main body of the markdown file?